<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendMessage</class>
 <widget class="QWidget" name="SendMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc.qrc">
    <normaloff>:/new/prefix1/image/writeMsg.png</normaloff>:/new/prefix1/image/writeMsg.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>发件人：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="ui_senderComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>收件人：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ui_recipient"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>抄送人：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ui_cc"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>密送人：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="ui_bcc"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>主  题：</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="ui_subject"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QFontComboBox" name="ui_fontComboBox"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="ui_sizeComboBox">
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>21</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>22</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>23</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>26</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>27</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>28</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>29</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="ui_BoldButton">
           <property name="toolTip">
            <string>加粗</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <disabledoff>:/new/prefix1/image/bold.png</disabledoff>
            </iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="ui_italicButton">
           <property name="toolTip">
            <string>斜体</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <disabledoff>:/new/prefix1/image/italic.png</disabledoff>
            </iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QToolButton" name="ui_leftButton">
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip">
            <string>左对齐</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <disabledoff>:/new/prefix1/image/leftAlignment.png</disabledoff>
            </iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QToolButton" name="ui_centerButton">
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip">
            <string>居中对齐</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <disabledoff>:/new/prefix1/image/centerAlignment.png</disabledoff>
            </iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QToolButton" name="ui_rightButton">
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>22</height>
            </size>
           </property>
           <property name="toolTip">
            <string>右对齐</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset>
             <disabledoff>:/new/prefix1/image/rightAlignment.png</disabledoff>
            </iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QToolButton" name="ui_addPicButton">
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../qrc.qrc">
             <normaloff>:/new/prefix1/image/addPic.png</normaloff>:/new/prefix1/image/addPic.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="10">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="8">
          <widget class="QToolButton" name="ui_attachButton">
           <property name="text">
            <string>附件</string>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <widget class="QToolButton" name="ui_addNameButton">
           <property name="text">
            <string>添加姓名</string>
           </property>
          </widget>
         </item>
         <item row="0" column="12">
          <widget class="QPushButton" name="ui_sendButton">
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item row="0" column="11">
          <widget class="QComboBox" name="ui_signature">
           <item>
            <property name="text">
             <string>-------我的签名------</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="ui_content"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc.qrc"/>
 </resources>
 <connections/>
</ui>
